lib/aruba/processes/basic_process.rb

```diff
@@ -115,7 +115,13 @@ def before_run; end
       def after_run; end
 
       def inspect
-        out = stdout(:wait_for_io => 0) + stderr(:wait_for_io => 0)
+        out = stdout(:wait_for_io => 0)
```

Comment:
If inspect is called before there's any stdout, they're `nil`, which causes adding them (`+`) to fail.

`inspect` methods shouldn't ever fail, or you can get a chain of failures of handling failures. Very confusing in practice, especially to contributors who do not expect things like `inspect` to every fail.